com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -1311,6 +1537,46 @@ internal static void SetRenderTarget(CommandBuffer cmd, RenderTargetIdentifier c
             }
         }
 
+        static void SetRenderTarget(CommandBuffer cmd,
+            RTHandle colorAttachment,
+            RenderBufferLoadAction colorLoadAction,
+            RenderBufferStoreAction colorStoreAction,
+            RTHandle depthAttachment,
+            RenderBufferLoadAction depthLoadAction,
+            RenderBufferStoreAction depthStoreAction,
+            ClearFlag clearFlags,
+            Color clearColor)
+        {
+            // XRTODO: Revisit the logic. Why treat CameraTarget depth specially?
```

Comment:
@thomas-zeng Can I get your input on this comment and https://github.com/Unity-Technologies/Graphics/pull/5346/files#diff-bb936e5ad76122e6fb0c811b99e0d39cfcb3ecd599323bd2436cdd75f7e42f1eR1528

Reply:
I think we can remove this comment. I believe the if branch is handling the case where depthAttachement is the same as colorAttachement.
The RenderTargetIdentifier version of SetRenderTarget above also sets depthLoadAction and depthStoreAction. Is there a specific reason why we don't set them in this version?

Reply:
The assumption was that if `if (depthAttachment.nameID == BuiltinRenderTextureType.CameraTarget)` is true, then there is no depth to bind. Is this a wrong assumption?
In any case, `CoreUtils.SetRenderTarget` would need a separate `RTHandle` for color and depth.